R/markdown.R

```diff
@@ -68,3 +68,15 @@ markdown_text <- function(text, ...) {
   write_lines(text, path = tmp)
   markdown(tmp, ...)
 }
+
+
+markdown_text2 <- function(text, ...) {
```

Comment:
This makes me a mildly worried. What happens if the user supplies a markdown that generates multiple paragraphs or a list or something more complicated?